packages/compiler/src/init/init.ts

```diff
@@ -19,7 +19,7 @@ import { isFileSkipGeneration, makeScaffoldingConfig, scaffoldNewProject } from
 export interface InitTypeSpecProjectOptions {
   readonly templatesUrl?: string;
   readonly template?: string;
-  readonly "auto-accept-prompts"?: boolean;
+  readonly "no-prompt"?: boolean;
```

Comment:
I think yargs also can do auto `no-` prefix for things, is it a pattern to have both `--prompt` and `--no-prompt`?

Reply:
Hmmm, `prompt` in particular feels unnecessary, though I don't mind adding for completeness.